mappings/net/minecraft/util/math/BlockPos.mapping

```diff
@@ -69,28 +69,89 @@ CLASS net/minecraft/class_2338 net/minecraft/util/math/BlockPos
 	METHOD method_10093 offset (Lnet/minecraft/class_2350;)Lnet/minecraft/class_2338;
 		ARG 1 direction
 	METHOD method_10094 iterate (IIIIII)Ljava/lang/Iterable;
+		ARG 0 startX
+		ARG 1 startY
+		ARG 2 startZ
+		ARG 3 endX
+		ARG 4 endY
+		ARG 5 endZ
 	METHOD method_10095 north ()Lnet/minecraft/class_2338;
 	METHOD method_10096 add (JIII)J
 		ARG 0 value
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
 	METHOD method_10097 iterate (Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;)Ljava/lang/Iterable;
-		ARG 0 pos1
-		ARG 1 pos2
+		ARG 0 start
+		ARG 1 end
 	METHOD method_17962 stream (IIIIII)Ljava/util/stream/Stream;
+		ARG 0 startX
+		ARG 1 startY
+		ARG 2 startZ
+		ARG 3 endX
+		ARG 4 endY
+		ARG 5 endZ
 	METHOD method_19438 deserialize (Lcom/mojang/datafixers/Dynamic;)Lnet/minecraft/class_2338;
 		ARG 0 dynamic
 	METHOD method_20437 stream (Lnet/minecraft/class_2338;Lnet/minecraft/class_2338;)Ljava/util/stream/Stream;
-		ARG 0 pos1
-		ARG 1 pos2
+		ARG 0 start
+		ARG 1 end
 	METHOD method_23627 stream (Lnet/minecraft/class_3341;)Ljava/util/stream/Stream;
 		ARG 0 box
 	METHOD method_25503 mutableCopy ()Lnet/minecraft/class_2338$class_2339;
+	METHOD method_25996 manhattanIterate (Lnet/minecraft/class_2338;III)Ljava/lang/Iterable;
```

Comment:
`iterateManhattan`. But I think an even better name would be `iterateOutwards`.

Reply:
```suggestion
	METHOD method_25996 iterateOutwards (Lnet/minecraft/class_2338;III)Ljava/lang/Iterable;
```